src/data/error_log.ts

```diff
@@ -7,3 +7,5 @@ export interface LogProvider {
 
 export const fetchErrorLog = (hass: HomeAssistant) =>
   hass.callApi<string>("GET", "error_log");
+
+export const getErrorLogDownloadUrl = () => "/api/error_log";
```

Comment:
```suggestion
export const getErrorLogDownloadUrl = "/api/error_log";
```

There is no logic here